WhiteDentalClinic/WhiteDentalClinic.Application/Services/Interfaces/IAppointmentService.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using WhiteDentalClinic.Application.Models.AppointmentModel;
+using WhiteDentalClinic.Application.Models.Dentist;
+
+namespace WhiteDentalClinic.Application.Services.Interfaces
+{
+    public interface IAppointmentService
+    {
+        IEnumerable<AppointmentResponseModel> GetAllAppointments();
```

Comment:
The fact that "Appointment" is repeated so many times in method names is a sign that maybe you should think about if it is redundant and can be removed. The answer is yes. 😄 You have the argument for it an a comment above.